### DISCUSSION TITLE
Reviewers per groups

### DISCUSSION THREAD

**Discussion by mancarv** [necojackarc/auto-request-review#53] [Q&A]
Hi, if we hace the option `number_of_reviewers: 2` and two groups, can it be configured to choose one reviewer per group?

**Comment by necojackarc** [comment]
Hi, @mancarv. That's a good question. I needed to browse the codebase to answer 😄 

The logic to pick reviewers randamly to the number specified in `number_of_reviewers` is here:

https://github.com/necojackarc/auto-request-review/blob/e08cdffa277d50854744de3f76230260e61c67f4/src/reviewer.js#L114-L124

And this is called at the end of the logic:

https://github.com/necojackarc/auto-request-review/blob/e08cdffa277d50854744de3f76230260e61c67f4/src/index.js#L65

That means, at this moment, it's not possible to configure the number of reviewers for each group. 

To implement this, we'd probably need to add another option (`number_of_reviewrs_per_group`, for instance).
